docs/guide/extending_ez_platform_ui.md

```diff
@@ -661,6 +661,77 @@ Optionally, Sub-items module can take a following list of props:
 The Online Editor is based on [Alloy Editor](https://alloyeditor.com/).
 Refer to [Alloy Editor documentation](https://alloyeditor.com/docs/develop/) to learn how to extend the Online Editor with new elements.
 
+### Custom tags
+
+Custom tags enable you to add more features to the Rich Text editor beyond the built-in ones.
+
+Custom tags are configured under the `ezrichtext` key (here on the example of a YouTube tag):
+
+``` yml
+ezpublish:
+    system:
+        default:
+            fieldtypes:
+                ezrichtext:
+                    custom_tags: [ezyoutube]
+
+    ezrichtext:
+        custom_tags:
+            ezyoutube:
+                template: 'AppBundle:field_type/ezrichtext/custom_tag:ezyoutube.html.twig'
+                icon: '/assets/field_type/ezrichtext/custom_tag/icon/youtube-color.svg#youtube-color'
+                attributes:
+                    title:
+                        type: 'string'
+                        required: true
+                        default_value: ''
+                    video_url:
+                        type: 'string'
+                        required: true
+                    width:
+                        type: 'number'
+                        required: true
+                        default_value: 640
+                    height:
+                        type: 'number'
+                        required: true
+                        default_value: 360
+                    autoplay:
+                        type: 'boolean'
+                        default_value: false
+                    align:
+                        type: 'choice'
+                        required: false
+                        default_value: 'left'
+                        choices: ['left', 'center', 'right']
+```
+
+Each custom tag can have any number of attributes. Supported attribute types are:
+`string`, `number`, `boolean` and `choice`.
```

Comment:
`choice` attribute requires list of choices provided by `choices` key. (_or something like that_ :grinning: )

Also worth to mention: if `required` key is omitted, it is assumed to be `false`